src/main/resources/templates/entries.html

```diff
@@ -11,11 +11,12 @@
       <div class="grid-row">
         <div class="column-two-thirds">
           <h1 class="heading-large">
+            <span th:include="fragments/pagination.html::pagination-top('entries')"></span>
             <span class="heading-secondary" th:if="${content.recordKey.present}">
-              Part of <a th:href="${'/record/' + content.recordKey.get()}" th:text="${'Record: ' + content.recordKey.get()}">Record: GM</a>
+              Part of Record:<a th:href="${'/record/' + content.recordKey.get()}" th:text="${content.recordKey.get()}"> GM</a>
```

Comment:
I think there's a whitespace issue here - shouldn't there be space between `Record:` and the primary key? ie `Record: foo` rather than `Record:foo`. (see posted screenshot: there's no space in "Part of Record:400673")